src/Control/HTTPResponse.php

```diff
@@ -310,6 +310,14 @@ public function removeHeader($header)
         return $this;
     }
 
+    /**
+     * Sanitise header values to avoid possible XSS vectors
+     */
+    private function sanitiseHeader(string $value): string
```

Comment:
This method is new from 4 and was conflicting with the strongly typed parameters of redirect in 5